androidApp/src/main/java/com/example/tasky/android/MainApplication.kt

```diff
@@ -19,6 +21,7 @@ class MainApplication : Application() {
     @OptIn(ExperimentalSettingsApi::class, ExperimentalSettingsImplementation::class)
     override fun onCreate() {
         super.onCreate()
+        createDatabase(getDatabaseBuilder(this))
```

Comment:
Would it make more sense here to create a new Koin module to handle the creation of the database? Something like this:

```
val databaseModule = module {
  single<TaskyDatabase> {
    Room.databaseBuilder(
      androidApplication(),
      TaskyDatabase::class.java,
      “tasky.db”
    ).build()
  }
}
```

Reply:
Becuase I am not going to access the database in the androidApp, but only in the shared/commonMain. is it still better to create it in Koin module?

Reply:
Yes, you want to be consistent in your project. It should be predictable where to look for DI-related code. Your `Application` class is responsible for wiring all your dependencies together, while each of your gradle modules (or packages) defines its own Koin module on how to create this gradle module’s dependencies.